examples/src/main/java/org/apache/ignite/examples/opencensus/OpenCensusMetricsExporterExample.java

```diff
@@ -31,7 +31,7 @@
 import org.apache.ignite.IgniteCache;
 import org.apache.ignite.Ignition;
 import org.apache.ignite.configuration.IgniteConfiguration;
-import org.apache.ignite.spi.metric.opencensus.OpenCensusMetricExporterSpi;
+import org.apache.ignite.opencensus.spi.metric.OpenCensusMetricExporterSpi;
```

Comment:
Why this change?

Reply:
Cause besides metrics tracing was added as one of OpenCensus related spis. So at the moment package tree is following:
```
-org.apache.ignite
  -opencensus.spi
    -metric
    -tracing
```

Reply:
This clashes with the existing package hierarchy.
We place spi's to `org.apache.ignite.spi.SPIPACKAGE`.

Let's keep current naming hierarchy.

Reply:
Should be fixed by now.